src/vs/platform/progress/common/progress.ts

```diff
@@ -51,6 +51,7 @@ export interface IProgressOptions {
 	total?: number;
 	cancellable?: boolean;
 	buttons?: string[];
+	command?: string;
```

Comment:
@alexr00 this seems to only be supported when progress location is status bar as such I suggest to introduce a similar concept as `IProgressNotificationOptions` to clarify and have that type be used when location is status.